docs/content/developer/iota-101/nft/marketplace.mdx

```diff
@@ -0,0 +1,218 @@
+---
+description: A brief introduction to implementing NFT marketplace extension using the Kiosk Apps standard in IOTA's Move language.
+---
+
+# Marketplace Extension Usage
+
```

Comment:
Could you add a brief introduction description to this page? In a similar fashion of what you find at the top of `rent-nft.mdx`

Reply:
Done